[I 2022-04-26 13:40:37.034 ServerApp] jupyterlab | extension was successfully linked.
[I 2022-04-26 13:40:37.043 ServerApp] nbclassic | extension was successfully linked.
[I 2022-04-26 13:40:42.205 ServerApp] notebook_shim | extension was successfully linked.
[I 2022-04-26 13:40:42.520 ServerApp] notebook_shim | extension was successfully loaded.
[I 2022-04-26 13:40:42.522 LabApp] JupyterLab extension loaded from /home/k/kimpson/bin/conda/envs/GPU/lib/python3.7/site-packages/jupyterlab
[I 2022-04-26 13:40:42.522 LabApp] JupyterLab application directory is /home/k/kimpson/bin/conda/envs/GPU/share/jupyter/lab
[I 2022-04-26 13:40:42.546 ServerApp] jupyterlab | extension was successfully loaded.
[I 2022-04-26 13:40:42.643 ServerApp] nbclassic | extension was successfully loaded.
[I 2022-04-26 13:40:42.644 ServerApp] Serving notebooks from local directory: /network/aopp/chaos/pred/kimpson/speedy-1
[I 2022-04-26 13:40:42.644 ServerApp] Jupyter Server 1.13.5 is running at:
[I 2022-04-26 13:40:42.644 ServerApp] http://localhost:9172/lab
[I 2022-04-26 13:40:42.644 ServerApp]  or http://127.0.0.1:9172/lab
[I 2022-04-26 13:40:42.644 ServerApp] Use Control-C to stop this server and shut down all kernels (twice to skip confirmation).
[I 2022-04-26 13:44:04.914 ServerApp] 302 GET / (::1) 1.15ms
[I 2022-04-26 13:44:04.935 LabApp] 302 GET /lab? (::1) 4.60ms
[W 2022-04-26 13:44:09.649 ServerApp] 401 POST /login?next=%2Flab%3F (::1) 2.63ms referer=http://localhost:8886/login?next=%2Flab%3F
[I 2022-04-26 13:44:12.573 ServerApp] 302 POST /login?next=%2Flab%3F (::1) 2.20ms
[W 2022-04-26 13:44:29.659 LabApp] Could not determine jupyterlab build status without nodejs
[W 2022-04-26 13:44:33.839 ServerApp] 404 GET /api/contents/reproducible_workflow/experiments/2.%20Lakes_with_extra_fields-Copy1.ipynb?content=0&1650977069585 (::1): No such file or directory: reproducible_workflow/experiments/2. Lakes_with_extra_fields-Copy1.ipynb
[W 2022-04-26 13:44:33.840 ServerApp] No such file or directory: reproducible_workflow/experiments/2. Lakes_with_extra_fields-Copy1.ipynb
[W 2022-04-26 13:44:33.841 ServerApp] 404 GET /api/contents/reproducible_workflow/experiments/2.%20Lakes_with_extra_fields-Copy1.ipynb?content=0&1650977069585 (::1) 6.38ms referer=http://localhost:8886/lab?
[W 2022-04-26 13:44:33.841 ServerApp] 404 GET /api/contents/reproducible_workflow/experiments/1.%20Lakes.ipynb?content=0&1650977069586 (::1): No such file or directory: reproducible_workflow/experiments/1. Lakes.ipynb
[W 2022-04-26 13:44:33.842 ServerApp] No such file or directory: reproducible_workflow/experiments/1. Lakes.ipynb
[W 2022-04-26 13:44:33.843 ServerApp] 404 GET /api/contents/reproducible_workflow/experiments/1.%20Lakes.ipynb?content=0&1650977069586 (::1) 6.88ms referer=http://localhost:8886/lab?
[W 2022-04-26 13:44:33.843 ServerApp] 404 GET /api/contents/reproducible_workflow/00.%20Workflow.ipynb?content=0&1650977069585 (::1): No such file or directory: reproducible_workflow/00. Workflow.ipynb
[W 2022-04-26 13:44:33.844 ServerApp] No such file or directory: reproducible_workflow/00. Workflow.ipynb
[W 2022-04-26 13:44:33.846 ServerApp] 404 GET /api/contents/reproducible_workflow/00.%20Workflow.ipynb?content=0&1650977069585 (::1) 8.86ms referer=http://localhost:8886/lab?
[W 2022-04-26 13:44:33.854 ServerApp] 404 GET /api/contents/reproducible_workflow/experiments/A1.V20_vs_V15.ipynb?content=0&1650977069586 (::1): No such file or directory: reproducible_workflow/experiments/A1.V20_vs_V15.ipynb
[W 2022-04-26 13:44:33.860 ServerApp] No such file or directory: reproducible_workflow/experiments/A1.V20_vs_V15.ipynb
[W 2022-04-26 13:44:33.862 ServerApp] 404 GET /api/contents/reproducible_workflow/experiments/A1.V20_vs_V15.ipynb?content=0&1650977069586 (::1) 8.97ms referer=http://localhost:8886/lab?
[W 2022-04-26 13:44:33.864 ServerApp] 404 GET /api/contents/reproducible_workflow/scripts/train_and_predict.py?content=0&1650977069587 (::1): No such file or directory: reproducible_workflow/scripts/train_and_predict.py
[W 2022-04-26 13:44:33.864 ServerApp] No such file or directory: reproducible_workflow/scripts/train_and_predict.py
[W 2022-04-26 13:44:33.865 ServerApp] 404 GET /api/contents/reproducible_workflow/scripts/train_and_predict.py?content=0&1650977069587 (::1) 2.11ms referer=http://localhost:8886/lab?
[W 2022-04-26 13:44:33.877 ServerApp] 404 GET /api/contents/reproducible_workflow/scripts/join_MODIS_with_ERA.py?content=0&1650977069587 (::1): No such file or directory: reproducible_workflow/scripts/join_MODIS_with_ERA.py
[W 2022-04-26 13:44:33.878 ServerApp] No such file or directory: reproducible_workflow/scripts/join_MODIS_with_ERA.py
[W 2022-04-26 13:44:33.879 ServerApp] 404 GET /api/contents/reproducible_workflow/scripts/join_MODIS_with_ERA.py?content=0&1650977069587 (::1) 3.47ms referer=http://localhost:8886/lab?
[W 2022-04-26 13:44:34.504 ServerApp] 404 GET /api/contents/reproducible_workflow/experiments?1650977074419 (::1): No such file or directory: reproducible_workflow/experiments
[W 2022-04-26 13:44:34.505 ServerApp] No such file or directory: reproducible_workflow/experiments
[W 2022-04-26 13:44:34.505 ServerApp] 404 GET /api/contents/reproducible_workflow/experiments?1650977074419 (::1) 2.87ms referer=http://localhost:8886/lab?
[W 2022-04-26 13:45:13.093 ServerApp] Notebook jupyter_notebooks/Fig1_temperature_timeseries.ipynb is not trusted
[I 2022-04-26 13:45:14.715 ServerApp] Kernel started: af773ec3-d5b5-4e97-a7d9-eb1c7a204802
[IPKernelApp] ERROR | No such comm target registered: jupyter.widget.control
[IPKernelApp] WARNING | No such comm: ede9c680-4f0c-4646-8e77-e8815df7cd19
[I 2022-04-26 13:46:00.625 ServerApp] Copying jupyter_notebooks/Fig1_temperature_timeseries-Copy1.ipynb to /jupyter_notebooks
[W 2022-04-26 13:46:00.763 ServerApp] Notebook jupyter_notebooks/Fig1_temperature_timeseries-Copy1.ipynb is not trusted
[W 2022-04-26 13:46:00.781 ServerApp] Notebook jupyter_notebooks/Fig1_temperature_timeseries-Copy2.ipynb is not trusted
[W 2022-04-26 13:46:02.240 ServerApp] Notebook jupyter_notebooks/100year.ipynb is not trusted
[I 2022-04-26 13:46:02.881 ServerApp] Kernel started: 78f7d50a-48ac-49d2-80ed-337936265ed5
[IPKernelApp] ERROR | No such comm target registered: jupyter.widget.control
[IPKernelApp] WARNING | No such comm: 9735de02-adc5-4f95-8a7b-a6c416081195
[I 2022-04-26 13:57:41.641 ServerApp] Creating new notebook in /jupyter_notebooks
[I 2022-04-26 13:57:42.241 ServerApp] Kernel started: 2dcc3a72-e834-4c19-99b5-8191139147ab
[IPKernelApp] ERROR | No such comm target registered: jupyter.widget.control
[IPKernelApp] WARNING | No such comm: de02c62d-db0f-41eb-b296-4ba2d53df918
[I 2022-04-26 13:59:42.239 ServerApp] Saving file at /jupyter_notebooks/1.Timeseries_100years_m6.ipynb
[I 2022-04-26 14:01:42.337 ServerApp] Saving file at /jupyter_notebooks/1.Timeseries_100years_m6.ipynb
[I 2022-04-26 14:03:42.446 ServerApp] Saving file at /jupyter_notebooks/1.Timeseries_100years_m6.ipynb
[I 2022-04-26 14:05:43.257 ServerApp] Saving file at /jupyter_notebooks/1.Timeseries_100years_m6.ipynb
[I 2022-04-26 14:07:43.356 ServerApp] Saving file at /jupyter_notebooks/1.Timeseries_100years_m6.ipynb
[I 2022-04-26 14:09:43.496 ServerApp] Saving file at /jupyter_notebooks/1.Timeseries_100years_m6.ipynb
[I 2022-04-26 14:09:43.590 ServerApp] Kernel interrupted: 2dcc3a72-e834-4c19-99b5-8191139147ab
[I 2022-04-26 14:09:44.068 ServerApp] Kernel interrupted: 2dcc3a72-e834-4c19-99b5-8191139147ab
[I 2022-04-26 14:11:44.265 ServerApp] Saving file at /jupyter_notebooks/1.Timeseries_100years_m6.ipynb
[I 2022-04-26 14:13:44.401 ServerApp] Saving file at /jupyter_notebooks/1.Timeseries_100years_m6.ipynb
[I 2022-04-26 14:17:44.562 ServerApp] Saving file at /jupyter_notebooks/1.Timeseries_100years_m6.ipynb
[I 2022-04-26 14:19:44.737 ServerApp] Saving file at /jupyter_notebooks/1.Timeseries_100years_m6.ipynb
[I 2022-04-26 14:29:46.341 ServerApp] Saving file at /jupyter_notebooks/1.Timeseries_100years_m6.ipynb
[I 2022-04-26 14:31:46.538 ServerApp] Saving file at /jupyter_notebooks/1.Timeseries_100years_m6.ipynb
[I 2022-04-26 14:33:46.724 ServerApp] Saving file at /jupyter_notebooks/1.Timeseries_100years_m6.ipynb
[I 2022-04-26 14:35:47.347 ServerApp] Saving file at /jupyter_notebooks/1.Timeseries_100years_m6.ipynb
[I 2022-04-26 14:37:48.633 ServerApp] Saving file at /jupyter_notebooks/1.Timeseries_100years_m6.ipynb
[I 2022-04-26 14:39:48.822 ServerApp] Saving file at /jupyter_notebooks/1.Timeseries_100years_m6.ipynb
[I 2022-04-26 14:41:49.071 ServerApp] Saving file at /jupyter_notebooks/1.Timeseries_100years_m6.ipynb
[I 2022-04-26 15:13:49.376 ServerApp] Saving file at /jupyter_notebooks/1.Timeseries_100years_m6.ipynb
[I 2022-04-26 15:15:49.619 ServerApp] Saving file at /jupyter_notebooks/1.Timeseries_100years_m6.ipynb
[I 2022-04-26 15:17:49.864 ServerApp] Saving file at /jupyter_notebooks/1.Timeseries_100years_m6.ipynb
[I 2022-04-26 15:19:50.107 ServerApp] Saving file at /jupyter_notebooks/1.Timeseries_100years_m6.ipynb
[I 2022-04-26 15:21:50.348 ServerApp] Saving file at /jupyter_notebooks/1.Timeseries_100years_m6.ipynb
[I 2022-04-26 15:23:50.599 ServerApp] Saving file at /jupyter_notebooks/1.Timeseries_100years_m6.ipynb
[I 2022-04-26 15:37:50.968 ServerApp] Saving file at /jupyter_notebooks/1.Timeseries_100years_m6.ipynb
[I 2022-04-26 15:39:51.309 ServerApp] Saving file at /jupyter_notebooks/1.Timeseries_100years_m6.ipynb
[I 2022-04-26 19:40:35.420 ServerApp] Starting buffering for 2dcc3a72-e834-4c19-99b5-8191139147ab:57deb11b-8a0c-4d27-b914-c9b47d3b2fbc
[I 2022-04-26 19:40:35.421 ServerApp] Starting buffering for af773ec3-d5b5-4e97-a7d9-eb1c7a204802:dd8ddd85-646c-43da-962d-95cda5f95033
[I 2022-04-26 19:40:35.421 ServerApp] Starting buffering for 78f7d50a-48ac-49d2-80ed-337936265ed5:c119a5f5-cd9c-4acd-b471-cb26c2f9516b
[I 2022-04-26 19:40:35.615 ServerApp] Restoring connection for 2dcc3a72-e834-4c19-99b5-8191139147ab:57deb11b-8a0c-4d27-b914-c9b47d3b2fbc
[I 2022-04-26 19:40:35.710 ServerApp] Restoring connection for 78f7d50a-48ac-49d2-80ed-337936265ed5:c119a5f5-cd9c-4acd-b471-cb26c2f9516b
[IPKernelApp] ERROR | No such comm target registered: jupyter.widget.control
[IPKernelApp] WARNING | No such comm: e410fb1d-9523-4a2a-ae07-65fdbd1008f6
[I 2022-04-26 19:40:35.788 ServerApp] Restoring connection for af773ec3-d5b5-4e97-a7d9-eb1c7a204802:dd8ddd85-646c-43da-962d-95cda5f95033
[IPKernelApp] ERROR | No such comm target registered: jupyter.widget.control
[IPKernelApp] WARNING | No such comm: 4059cc3d-8c86-4e56-8850-3bf87fca7b9b
[I 2022-04-26 21:30:13.768 ServerApp] Starting buffering for 2dcc3a72-e834-4c19-99b5-8191139147ab:57deb11b-8a0c-4d27-b914-c9b47d3b2fbc
[I 2022-04-26 21:30:13.769 ServerApp] Starting buffering for 78f7d50a-48ac-49d2-80ed-337936265ed5:c119a5f5-cd9c-4acd-b471-cb26c2f9516b
[I 2022-04-26 21:30:13.770 ServerApp] Starting buffering for af773ec3-d5b5-4e97-a7d9-eb1c7a204802:0d7b64aa-2871-4755-9498-aae7c2e8d762
[IPKernelApp] ERROR | No such comm target registered: jupyter.widget.control
[IPKernelApp] WARNING | No such comm: 50c4bcfa-e23b-45f0-90be-a8e532690433
[I 2022-04-26 21:30:13.908 ServerApp] Restoring connection for 2dcc3a72-e834-4c19-99b5-8191139147ab:57deb11b-8a0c-4d27-b914-c9b47d3b2fbc
[I 2022-04-26 21:30:13.982 ServerApp] Restoring connection for 78f7d50a-48ac-49d2-80ed-337936265ed5:c119a5f5-cd9c-4acd-b471-cb26c2f9516b
[IPKernelApp] ERROR | No such comm target registered: jupyter.widget.control
[IPKernelApp] WARNING | No such comm: 7149d23a-6a78-48f0-8e13-4f1beac44308
[W 2022-04-26 21:33:13.843 ServerApp] WebSocket ping timeout after 119960 ms.
[I 2022-04-26 21:33:13.847 ServerApp] Starting buffering for 78f7d50a-48ac-49d2-80ed-337936265ed5:c119a5f5-cd9c-4acd-b471-cb26c2f9516b
[I 2022-04-26 21:33:13.848 ServerApp] Starting buffering for 2dcc3a72-e834-4c19-99b5-8191139147ab:57deb11b-8a0c-4d27-b914-c9b47d3b2fbc
[I 2022-04-26 21:33:13.848 ServerApp] Starting buffering for af773ec3-d5b5-4e97-a7d9-eb1c7a204802:dd8ddd85-646c-43da-962d-95cda5f95033
